### DISCUSSION TITLE
Where to give AlDente 5 stars 🌟🌟🌟🌟🌟

### DISCUSSION THREAD

**Discussion by gordoncy** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#537] [General]
Well done. This is the application I have been waiting for for years. More people should know about this.
Where can I go to give a 5 star review 🌟🌟🌟🌟🌟? 
It's not in the Apple App Store, but there must be a place that users can give it a thumb up👍?

**Comment by MatthiasKerbl** [comment]
> Well done. This is the application I have been waiting for for years. More people should know about this. Where can I go to give a 5 star review 🌟🌟🌟🌟🌟? It's not in the Apple App Store, but there must be a place that users can give it a thumb up👍?

Hi @gordoncy,

Thank you, we really appreciate that you like our little app AlDente.
If you want to support us you can write us a review on Trustpilot: https://www.trustpilot.com/review/apphousekitchen.com

Thanks a lot,
David and Matthias
